oxidation/client/src/views/HomePage.vue

```diff
@@ -319,7 +334,7 @@ function onOrganizationCardClick(device: AvailableDevice): void {
   selectedDevice = device;
 }
 
-async function login(): Promise<void> {
+async function login(workspace: string): Promise<void> {
```

Comment:
What's this `workspace` variable?

Reply:
I think it's because we need to know which workspace we are in. 
@NicoTuxx Do I am right?

Reply:
This function is called by the click event on the button,
```
 <ion-button
     @click="login($event)"
```
I have no idea what this `$event` is or what purpose it serves. It's probably better to edit the `@click` event to remove the $event and to remove the argument workspace from the login function. Workspaces don't appear until after we're logged in anyway.

Reply:
$event is here anyway, @click is a default vue eventListener we can't edit him to remove the param.
You can choose to use $event or not by passing it or not to your callback function, that's all.
In this case, we don't use it in the callback function so I removed it.

On the other side, 'workspace' parameter seems to be an obsolete residue of ancient code. Removed it.